zmon-controller-ui/views/checkDefinitionView.html

```diff
@@ -8,6 +8,7 @@ <h1>Check: {{checkDefinition.name}}</h1>
         <p>
             ID: {{checkDefinition.id}},
             Status: <status status="checkDefinition.status"/>,
+            Tier: {{checkDefinition.tier}},
```

Comment:
do we want to show it if it's unknown

Reply:
Yes, to help users complain more specifically.

Reply:
But as @alexkorotkikh suggests, all unknowns will be showed as `other`

Reply:
Now it will be shown as "normal"